devops/scripts/create-sycl-user.sh

```diff
@@ -13,7 +13,10 @@ USER_ID=1001
 groupadd -g $USER_ID $USER_NAME && useradd $USER_NAME -u $USER_ID -g $USER_ID -m -s /bin/bash
 # Add user to video/irc groups so that it can access GPU
 usermod -aG video $USER_NAME
-usermod -aG irc $USER_NAME
+# Not all OS have irc group
+if grep -qi 'ubuntu' /etc/os-release; then
```

Comment:
nit: can we instead grep `/etc/group` for `irc`? Slightly more robust than an OS check